### ISSUE TITLE
Remove benchmarks

### ISSUE THREAD

**Issue by stas** [fast-jsonapi/fast_jsonapi#69]
## Expected Behavior

I removed the benchmarks as part of the #40 to cleanup the tests runtime.
We should remove them completely since it's not really relevant to keep them as part of the repository.

## Actual Behavior

Ideally we'd improve an independently maintained set of benchmarks and comparisons.
I started the project here:
https://github.com/fast-jsonapi/comparisons

**Comment by stas** [comment]
Example of the comparison (just for `fast_jsonapi`):
```
/app # benchmark-driver ./all.yml --bundler
Warming up --------------------------------------
            fast_jsonapi    59.583k i/s -     60.032k times in 1.007541s (16.78μs/i)
      fast_jsonapi(many)     75.615 i/s -      78.000 times in 1.031542s (13.22ms/i)
   fast_jsonapi(include)     20.632 i/s -      22.000 times in 1.066320s (48.47ms/i)
active_model_serializers     1.062M i/s -      1.114M times in 1.049834s (942.00ns/i)
    jsonapi-serializable     1.117M i/s -      1.177M times in 1.054233s (895.52ns/i)
     jsonapi-serializers     1.121M i/s -      1.164M times in 1.038117s (891.88ns/i)
Calculating -------------------------------------
            fast_jsonapi    56.977k i/s -    178.748k times in 3.137201s (17.55μs/i)
      fast_jsonapi(many)     73.293 i/s -     226.000 times in 3.083532s (13.64ms/i)
   fast_jsonapi(include)     20.153 i/s -      61.000 times in 3.026777s (49.62ms/i)
active_model_serializers     1.031M i/s -      3.185M times in 3.090062s (970.28ns/i)
    jsonapi-serializable     1.140M i/s -      3.350M times in 2.938390s (877.12ns/i)
     jsonapi-serializers
     1.159M i/s -      3.364M times in 2.902290s (862.83ns/i)

Comparison:
     jsonapi-serializers:   1158980.5 i/s 
    jsonapi-serializable:   1140088.8 i/s - 1.02x  slower
active_model_serializers:   1030626.6 i/s - 1.12x  slower
            fast_jsonapi:     56976.9 i/s - 20.34x  slower
      fast_jsonapi(many):        73.3 i/s - 15813.07x  slower
   fast_jsonapi(include):        20.2 i/s - 57507.79x  slower

```

**Comment by stas** [comment]
Fixed in #70
